sdk/attestation/Azure.Security.Attestation/tests/Samples/AttestationServiceAttestationSamples.cs

```diff
@@ -116,15 +116,18 @@ public async Task AttestingAnSgxEnclave()
             var client = GetAttestationClient();
 
             IReadOnlyList<AttestationSigner> signingCertificates = (await client.GetSigningCertificatesAsync()).Value;
+
+            #region Snippet:AttestSgxEnclave
             {
```

Comment:
We don't - stylistically I prefer the introduction of scopes across logical operations because it helps with managing variable lifetimes (I've gotten burned by type declarations leaking across long spans of code). Fixing.